ios/config.json

```diff
@@ -18,5 +18,10 @@
     "HLLocalization.bundle",
     "HLResources.bundle",
     "KonotorModels.bundle"
-  ]
+  ],
+  "plist": {
+    "NSMicrophoneUsageDescription": "This game uses the microphone, so you can send us voice feedback",
+    "NSPhotoLibraryUsageDescription": "This game uses the photos, so you can send us pictures of the problem you are facing",
+    "NSCameraUsageDescription": "This game uses the camera, so you can send us pictures of the problem you are facing"
```

Comment:
hotline takes access to the Camera because when uploading a picture, the 2 options that come up are taking photos from the library/from the camera

We would have to ask hotline to remove it